Formula/pysvn.rb

```diff
@@ -0,0 +1,70 @@
+class Pysvn < Formula
+  desc "Python interface to Subversion"
+  homepage "http://pysvn.tigris.org/"
+  url "http://pysvn.barrys-emacs.org/source_kits/pysvn-1.8.0.tar.gz"
+  sha256 "39596f4884ed689cdb5a4e210e421724302a566c7ba756cc4d46bbfeb0c8326b"
+
+  depends_on "openssl"
+  depends_on "subversion"
+  depends_on :python => :recommended
```

Comment:
A good point @xu-cheng; I just noticed the `test` so seems this should be in homebrew/python, yeh.